promoted_python_delivery_client/client/client.py

```diff
@@ -108,10 +116,13 @@ def _log_to_metrics(self,
                         cohort_membership: Optional[CohortMembership],
                         exec_svr: ExecutionServer) -> None:
         log_request = self._create_log_request(delivery_request, response, cohort_membership, exec_svr)
-        try:
-            self.api_metrics.run_metrics_logging(log_request)
-        except Exception as ex:
-            logging.error("Error logging to metrics", exc_info=ex)
+        if self.only_log_metrics_request:
```

Comment:
I assumed `only_log_metrics_request` would have logged to the Metrics API log method.  Maybe rename this field to "text_log_metrics_request".

Reply:
Unfortunately "log" is overloaded I guess. I will find some different name.